dossierdata.v1.openapi.yaml

```diff
@@ -9840,6 +9855,14 @@ components:
       example: "1"
       schema:
         type: integer
+    UserDepartmentId:
+      name: userDepartment_id
```

Comment:
Dit moet gelijk zijn aan de param in de url `user_department_id`

![image](https://user-images.githubusercontent.com/4311594/111986852-bfe21580-8b0e-11eb-8ded-e38750214c5f.png)